atomicapp/nulecule/main.py

```diff
@@ -147,10 +147,19 @@ def genanswers(self, dryrun=False, answers_format=None, **kwargs):
         self.answers_format = answers_format or ANSWERS_FILE_SAMPLE_FORMAT
 
         # Check to make sure an answers.conf file doesn't exist already
-        answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        if location is None:
+            answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        else:
+            answers_file = os.path.join(location, ANSWERS_FILE)
```

Comment:
sigh.. another reason to retire :) 

The link i used was based on master.. which moved since I pasted the link.. lets try using a link where the content won't move:
https://github.com/projectatomic/atomicapp/blob/fdc8a0bfb691d9117c5d1bd78549a37a84e39b74/atomicapp/nulecule/main.py#L76

basically if the path is absolute then we want to convert to the equivalent path inside the container. we are using the `getRoot()` function for this in other places.